src/test/app/claims/converters/claimantResponseConverter.ts

```diff
@@ -160,14 +160,16 @@ describe('claimant response converter', () => {
         draftClaimantResponse.accepted = new ClaimSettled(YesNoOption.NO)
         draftClaimantResponse.partPaymentReceived = new PartPaymentReceived(YesNoOption.YES)
         draftClaimantResponse.freeMediation = new FreeMediation('yes')
+        draftClaimantResponse.rejectionReason = new RejectionReason('OBJECTION!')
 
         expect(converter.convertToClaimantResponse(claim, draftClaimantResponse, mediationDraft, false)).to.deep.eq({
-          'type': 'REJECTION',
-          'freeMediation': 'yes',
-          'mediationContactPerson': undefined,
-          'mediationPhoneNumber': '07777777788',
-          'paymentReceived': 'yes',
-          'settleForAmount': 'no'
+          type: 'REJECTION',
+          freeMediation: 'yes',
+          mediationContactPerson: undefined,
+          mediationPhoneNumber: '07777777788',
+          paymentReceived: 'yes',
+          settleForAmount: 'no',
+          reason: 'OBJECTION!'
```

Comment:
It's a json object, so i prefer to put quotes around keys. Also, it will be consistent with other tests in the file.

Reply:
sure!